packages/language-service/test/language-plugin.js

```diff
@@ -16,7 +16,9 @@ test('create virtual code w/ mdxjsEsm', () => {
 
   const snapshot = snapshotFromLines('import {Planet} from "./Planet.js"', '')
 
-  const code = plugin.createVirtualCode?.('/test.mdx', 'mdx', snapshot)
+  const code = plugin.createVirtualCode?.('/test.mdx', 'mdx', snapshot, {
+    getAssociatedScript: () => undefined
```

Comment:
I suppose this is fine, but what is it for?

Reply:
This comes from https://github.com/volarjs/volar.js/pull/194, `createVirtualCode` belongs to internal API so we did not make `getAssociatedScript` optional. Since MDX does not have multiple sources, just return undefined.